client/driver/lxc.go

```diff
@@ -250,6 +270,13 @@ func (d *LxcDriver) Start(ctx *ExecContext, task *structs.Task) (*StartResponse,
 		fmt.Sprintf("%s alloc none rw,bind,create=dir", ctx.TaskDir.SharedAllocDir),
 		fmt.Sprintf("%s secrets none rw,bind,create=dir", ctx.TaskDir.SecretsDir),
 	}
+
+	for _, volDesc := range driverConfig.Volumes {
```

Comment:
Please guard this in a similar fashion to how it is done in Docker: https://github.com/hashicorp/nomad/blob/master/client/driver/docker.go#L85
https://github.com/hashicorp/nomad/blob/master/client/driver/docker.go#L496
https://github.com/hashicorp/nomad/blob/master/client/driver/docker.go#L1003